Bottom: 93a37342f9ffc0c764c93e4e06dd84960dfe45c7
Top:    6c68d208273b143e17de8aff5ebfff6c80a36e25
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 13:30:34 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/lib/master.c b/lib/master.c
index 96f2681..32cadf2 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1349,7 +1349,7 @@ static void write_map(char *name, struct mapent *first)
 		printf("  map file not created.\n")
 		return;
 	}
- 
+
 	do {
 		fprintf(f, "%s\t%s\n", me->key, me->mapent);
 	} while ((me = cache_lookup_next(first->mc, me)));
